src/App.jsx

```diff
@@ -26,8 +26,14 @@ function App() {
             <Route path="/" element={<Home />}></Route>
             <Route path="/topics" element={<Topics />}></Route>
             <Route path="/users" element={<Users />}></Route>
-            <Route path="/articles/:article_id" element={<SingleArticle currentUser={currentUser}/>}></Route>
-            <Route path="/articles/:article_id/comments" element={<ArticleComments />}></Route>
+            <Route
+              path="/articles/:article_id"
+              element={<SingleArticle currentUser={currentUser} />}
+            ></Route>
+            <Route
```

Comment:
as the single article component is already rendering the comments for that article below it, we don't need to have a separate page accessible that displays only the comments which is what is happening at /articles/:article_id/comments - a user would probably not navigate to this url as we don't have any links in our app to it. In our FE side, we don't need to have the same urls as endpoints in the api - if that doesn't make sense, let me know and we can have a chat about it! :)

Reply:
Thanks Saima, that makes perfect sense. I'll take that out as its unnecessary and a duplicate